halyard-web/src/main/java/com/netflix/spinnaker/halyard/model/v1/providers/Provider.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.model.v1.providers;
+
+import java.util.List;
+
+public class Provider<T extends Account> {
+  public boolean enabled;
```

Comment:
Is there any particular reason you're using public values instead of private + getters/setters?

If not, I strongly recommend private vals + Lombok's `@Data`.